README.md

```diff
@@ -206,11 +206,11 @@ async function main() {
 main();
 ```
 
-The argument to `quicktype` is a complex object with many optional properties. [Explore its definition](https://github.com/quicktype/quicktype/blob/javascript-use-examples/src/quicktype-core/Run.ts#L171) to understand what options are allowed.
+The argument to `quicktype` is a complex object with many optional properties. [Explore its definition](https://github.com/quicktype/quicktype-vscode/blob/master/src/extension.ts#L141) to understand what options are allowed.
```

Comment:
Isn't the definition in quicktype-vscode only a subset of total functionality?
[The main entry point](https://github.com/quicktype/quicktype/blob/master/src/quicktype-core/Run.ts#L119) has more options to offer.

Reply:
@skornel02 you are absolutely right, I've just learned it today after digging into the project. The learning curve here is pretty steep 😅

Reply:
changed